remoteworkitem/tracker.go

```diff
@@ -5,7 +5,7 @@ import "github.com/almighty/almighty-core/models"
 // Tracker represents tracker configuration
 type Tracker struct {
 	models.Lifecycle
-	ID uint64 `gorm:"primary_key"`
+	ID string `gorm:"primary_key"`
```

Comment:
@aslakknutsen ah i confused everything, thanks for clearing, changing the required fields.